sympy/core/basic.py

```diff
@@ -1037,7 +1037,7 @@ def xreplace(self, rule):
         >>> x, y, z = symbols('x y z')
         >>> (1 + x*y).xreplace({x: pi})
         pi*y + 1
-        >>> (1 + x*y).xreplace({x:pi, y:2})
+        >>> (1 + x*y).xreplace({x: pi, y: 2})
```

Comment:
Somehow this looks odd to me, but I'm not sure whether it's SymPy code that's consistently using `{x:pi, y:2}`, or whether I saw that elsewhere (which would then be irrelevant).
Do you have a way to find out how the dict displays in SymPy are commonly written? I.e. how many with a space after the colon, and how many without a space after the colon?